src/components/assets/Asset.css

```diff
@@ -1,22 +1,43 @@
 .asset-container {
   display: flex;
   justify-content: flex-start;
+  flex-direction: column;
 }
 
 .preview {
-  margin-right: 25px;
+  margin: 50px auto;
 }
 
 .preview a {
   border-bottom: none;
 }
 
 .metadata {
-  width: 60%;
-  overflow-wrap: break-word;
+  width: 100%;
+  background-color: white;
+  border-top: 1px solid #DBE3E7;
+  border-bottom: 1px solid #DBE3E7;
+  display: flex;
+  padding: 40px 20px;
+  justify-content: flex-start;
   composes: field from '../entries/Field.css'
 }
-
+.metadata section {
+  overflow-wrap: break-word;
+  width: 50%;
+  padding:20px;
+}
+.metadata section div {
```

Comment:
it might be good to use child selectors here, such as `.metadata > section > div` or even just `.metadata section > div`. `div` (and `span`) is such a generic selector that it can cause trouble in the future if adding more structure down the tree and not understanding where styling is coming from. Or worse, depending on that style and needing to change that someday.